src/lightning_habana/__about__.py

```diff
@@ -1,10 +1,10 @@
-__version__ = "0.1.0rc1"
+__version__ = "1.10.0.0"
```

Comment:
What is this version related too, it is not semantic nor matching lightning

Reply:
this is to match hpu releases. So that hpu support can be mapped across releases with lightning

Reply:
so we do not expect any added features/API for HPU 1.10.0
I would suggest following semantic versioning and if needed adding a compatibility matrix